engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -33,7 +32,7 @@
     void onChunkUnloaded(Vector3i chunkPos);
 
     public enum WorldRenderingStage {
-        DEFAULT,
+        MONO,
         OCULUS_LEFT_EYE,
         OCULUS_RIGHT_EYE
```

Comment:
I can test Oculus stuff, but yeah, tricky to work that incrementally without being able to test locally